src/dd/Complex.cpp

```diff
@@ -29,6 +29,16 @@ bool Complex::approximatelyZero() const noexcept {
   return RealNumber::approximatelyZero(r) && RealNumber::approximatelyZero(i);
 }
 
+void Complex::mark() const noexcept {
+  r->mark();
+  i->unmark();
+}
+
+void Complex::unmark() const noexcept {
+  r->mark();
+  i->unmark();
+}
```

Comment:
```suggestion
void Complex::mark() const noexcept {
  r->mark();
  i->mark();
}

void Complex::unmark() const noexcept {
  r->unmark();
  i->unmark();
}
```
I suppose this was merely an oversight

Reply:
Yes. Thanks 🤦🏻